apps/emqx_bridge/src/emqx_bridge.erl

```diff
@@ -115,12 +116,12 @@ load_hook(Bridges) ->
         maps:to_list(Bridges)
     ).
 
-do_load_hook(Type, #{local_topic := _}) when ?EGRESS_DIR_BRIDGES(Type) ->
+do_load_hook(Type, #{local_topic := LocalTopic}) when
+    ?EGRESS_DIR_BRIDGES(Type) andalso is_binary(LocalTopic)
+->
```

Comment:
There may be multiple bridges enabled in an application.
Some of them may not have `local_topic` configuration, so it may not be necessary to determine whether to load in the `load_hook` stage.
The issue should be fixed in `e5.0.0` instead of postponed to `e5.0.1`.
see also #9686

Reply:
Apparently fixed in https://github.com/emqx/emqx/pull/9687